### ISSUE TITLE
+ Page "Why not markdown?"

### ISSUE THREAD

**Issue by sethwoodworth** [gitenberg-dev/documentation#1]
Make a page describing why we're not using markdown.

**Comment by samwilson** [comment]
Also, "why not ReST"?

**Comment by eshellman** [comment]
More precisely, we're not _focusing_ our development on a markdown-based toolchain. (or LaTeX, or TEI). I hope we don't do anything that would prevent addition of these in the future. For example, Gitbook has done some nice work on Markdown- we're borrowing ideas and I hope it goes in both directions.

**Comment by samwilson** [comment]
Ah right. Yes, that sounds sensible.

By the way, is this documentation repo going to be included in the main gitenberg.org site somehow? I'm never very sure how relative links are meant to work within Github.

**Comment by sethwoodworth** [comment]
Agreed.  There are too many high quality books from Distributed Proofreaders in LaTeX, TEI and even ReST to refuse to use them.

One of the benefits of the current travis-ci build system is that we can configure how to create an epub file on a repo-by-repo basis.

**Comment by sethwoodworth** [comment]
Yes @samwilson This repo is intended to be used on the website.  But it hasn't happened yet and I haven't decided how I'm going to implement it.

We could also include everything into a single file and turn it into an ebook!

**Comment by samwilson** [comment]
Cool! Yeah, I've been wondering if there would be a central build system, or if individual repos could specify their own. If it's Travis, then it's easy to build whatever custom bits need it; that's great!

I'm going to try adding a Travis config to one of the books I'm working on. (Or, maybe the documentation repo itself?)

:-)

**Comment by sethwoodworth** [comment]
Take a look at [Huck Finn's travis file](https://github.com/GITenberg/Adventures-of-Huckleberry-Finn_76/blob/c3b5cbfda913752b82a7644d3aadc5c8944d689f/.travis.yml)

**Comment by samwilson** [comment]
Just for interest: Pandoc now supports TEI output. https://github.com/jgm/pandoc/releases/tag/1.17
